core/trino-main/src/main/java/io/trino/server/security/oauth2/OAuth2ServiceModule.java

```diff
@@ -43,6 +43,7 @@ protected void setup(Binder binder)
         jaxrsBinder(binder).bind(OAuth2CallbackResource.class);
         newOptionalBinder(binder, OAuth2WebUiInstalled.class);
         newOptionalBinder(binder, OAuth2TokenExchange.class);
+        newOptionalBinder(binder, OAuth2TokenHandler.class);
```

Comment:
If you make OAuth2TokenHandler not Optional, then remove this line

Reply:
👍